search_params.go

```diff
@@ -73,8 +73,13 @@ func NewSearchParams(idx Index, params json.RawMessage, sel *C.FaissIDSelector,
 		if ivfParams.NprobePct > 0 {
 			nlist := float32(C.faiss_IndexIVF_nlist(ivfIdx))
 			nprobe = int(nlist * (ivfParams.NprobePct / 100))
+			if nprobe == 0 {
```

Comment:
Think you can replace this if condition with something along the lines of:
```
nprobe = max(int(nlist * (ivfParams.NprobePct / 100)), 1)
```
This will pick 1 if the value of nprobe is 0.

Reply:
Ack.